assets/z_app.css

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2024 D-Wave
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/* Style rules for all custom CSS. */
+
+.tab-content-results {
+    padding: 0 3rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+}
+
+.solution-stats-table td:nth-child(2n) {
+    text-align: right;
+}
+
+.solution-stats-table td:nth-child(2n+1) {
+    border-right: none;
+}
+
+.input, .results {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+}
```

Comment:
Why not put these in one of the other CSS files and keep this only for user custom CSS?

Reply:
I assume they'll want to get rid or alter most of these rules

Reply:
I would think most would just keep the solution stats table as is, just adding relevant data to it. I reckon it'd be easier to have these (or at least some of these) as default in another file and then have the user overwrite them here if needed. Also, comments above each block in this file describing what it changes and why it should/could be removed (if relevant) would be nice.

Reply:
These 4 rules are just to make the placeholder content look good so I expect most if not all will be removed or changed. I've added a comment that hopefully makes this more clear, let me know if this is better https://github.com/dwave-examples/template-dash/pull/2/commits/9c790f0ea295d743bf94ecbdeec9ee1e7b21a473